demisto_sdk/commands/common/hook_validations/pack_unique_files.py

```diff
@@ -611,6 +639,39 @@ def _is_approved_tags(self) -> bool:
                 return False
         return True
 
+    def filter_by_marketplace(self, marketplaces):
+        """Filtering pack_metadata tags by marketplace"""
+        pack_meta_file_content = self._read_metadata_content()
+
+        pack_tags: Dict[str, List[str]] = {}
+        for marketplace in marketplaces:
+            pack_tags[marketplace] = []
+        pack_tags['common'] = []
+
+        try:
+            for tag in pack_meta_file_content.get('tags', []):
+                if ':' in tag:
+                    tag_data = tag.split(':')
+                    tag_marketplaces = tag_data[0].split(',')
+                    for tag_marketplace in tag_marketplaces:
+                        pack_tags[tag_marketplace].append(tag_data[1])
+                else:
+                    pack_tags['common'].append(tag)
+        except KeyError:
+            print_warning('You have non-approved tag prefix in the pack metadata tags, cannot validate tags until it is fixed.')
```

Comment:
Print the list of approved tags (or link to it), as well as the tags.

Reply:
Will add